### DISCUSSION TITLE
Color shift after transforming color space

### DISCUSSION THREAD

**Discussion by rgdcastro** [dlemstra/Magick.NET#1192] [Help]
Hi there,

I am trying to convert a JPG image with **USWebCoatedSWOP** color profile to **AdobeRGB1998**.

However, there is a bit of a color shift when I try to use the following function,
`_image.TransformColorSpace(ColorProfile.USWebCoatedSWOP, ColorProfile.AdobeRGB1998);`

I have attached a screenshot of the images in Photoshop. You'll see that the black parts became somewhat grayish.

**Original**:
![Original](https://user-images.githubusercontent.com/36554808/171932950-7ab965b8-b409-42bd-ada4-5a81d66571f1.png)

**Converted**:
![Converted](https://user-images.githubusercontent.com/36554808/171932981-22dbc93d-543c-4ea5-91a8-cbd6a1d1cd8a.png)

I have tried doing the conversion on Adobe Photoshop and that doesn't seem to happen there.

Am I missing something?

**Comment by dlemstra** [comment]
Is your input image a CMYK image and can you share the image?
